app/controllers/skills/new.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.ObjectController.extend({
+  needs: ['skills'],
+  // replace this with data from rails api call (get all groups)
+  groups: ["1407", "1409", "1410", "1412"],
+  featured: ['True', 'False'],
+
+  actions: {
+    saveSkill: function () {
+      this.store.createRecord('skill', {
+        name: this.get('name'),
+        group: this.get('group'),
+        featured: this.get('featured'),
+      }).save().then(function () {
```

Comment:
Not totally clear on this, but I wonder if the willTransition action makes sense here. Looks like you're saying "Save this, then call a function that transitions to skills/index. Maybe there are some validations happening on the rails side that will kick back an error, but I think willTransition is built for this purpose in ember. The docs say a good time to use this pattern is "preventing navigation when a form is half-filled out."

If Rails is handling errors in the form via validations, that's cool. I'd be interested to see what happens if we leave a name out of the form and submit it though (in terms of transition). 

Otherwise, I like that I understand what I'm seeing here. It seems like such a common pattern that I wonder if there's an ember built in that's more succinct (e.g. save, redirect)